test/unit-tests/broker-launch.test.js

```diff
@@ -8,23 +8,28 @@ const launchBroker = require('../../dashboard/index');
 // mock event emitter to simulate ChildProcess object
 const mockChildProcess = new EventEmitter();
 
-// sinon fakes for behaviour verification
-let fake = sinon.fake();
-let spawnArgs;
+// sinon spies for behaviour verification
+let unrefSpy = sinon.spy();
+let disconnectSpy = sinon.spy();
+
+// storing stubbed methods arguments for verification
+let spawnArgs, processArgs;
 
 describe('Broker daemonized launching', () => {
     before(() => {
         // stub cp.spawn, to avoid actual daemon launch
         sinon.stub(cp, 'spawn').callsFake((...args) => {
             spawnArgs = args;
```

Comment:
Use the stub itself to validate the args. See [comment](https://github.com/elit-altum/newman-dashboard/pull/7/files/a9f818fe5f8167a89e41fd6abf30d581e6faa8ea..faf2d6e21f9aa287f034a94f73a6630f6b2fdeb5#r652537650) below.